compiler/ballerina-lang/src/main/resources/compiler.properties

```
@@ -1138,6 +1138,9 @@ error.cannot.assign.value.to.constant=\
 error.cannot.define.constant.with.type=\
   constant cannot be defined with type ''{0}'', expected a simple basic types or a map of a simple basic type
 
+error.cannot.define.constant.with.identifier=\
+  constant cannot be defined with identifier ''{0}''
```

Again, if we are going with a new error I think we can change this to something like
```suggestion
  constant cannot be defined with ''_''
```

since ''{0}'' will always be ''_''